### ISSUE TITLE
vot train dataset

### ISSUE THREAD

**Issue by ShangGaoG** [lawrence-cj/ARKitTrack#6]
Hi, thank you for your outstanding contribution. Is the image‘s naming format in the vot.zip the final version?

**Comment by lawrence-cj** [comment]
Yes, correctOn Sep 14, 2023, at 23:59, ShangGaoG ***@***.***> wrote:﻿
Hi, thank you for your outstanding contribution. Is the image‘s naming format in the vot.zip the final version?

**Comment by ShangGaoG** [comment]
Thanks for your reply. But it looks like the code provided can't read this naming format correctly, like 760577.250899.jpg. By the way, why are there multiple truth box files, like "groundtruth_1.txt, groundtruth_2.txt'?

**Comment by lawrence-cj** [comment]
I’ll check the code. Different truth box indicates different object. There may be multiple objects in one video. On Sep 15, 2023, at 10:18, ShangGaoG ***@***.***> wrote:﻿
Thanks for your reply. But it looks like the code provided can't read this naming format correctly, like 760577.250899.jpg. By the way, why are there multiple truth box files, like "groundtruth_1.txt, groundtruth_2.txt'?
